modules/sdk-coin-ton/src/lib/transaction.ts

```diff
@@ -28,7 +30,9 @@ export class Transaction extends BaseTransaction {
 
   constructor(coinConfig: Readonly<CoinConfig>) {
     super(coinConfig);
-    this.bounceable = false;
+    this.bounceable = true;
```

Comment:
will there be instances of types of transactions where bouncable is false?

Reply:
Yes, we want to be able to build transactions with bounceable as false for our new v1 wallets that we are planning to roll out for TON. For older wallets, bounceable remains true by default, and for newer wallets, we will be setting this flag as needed.